### DISCUSSION TITLE
Link al QMP3

### DISCUSSION THREAD

**Discussion by RaniAgus** [dds-jv/foro#365] [Consultas generales] [answered]
¡Buenas! Solo quería avisar que en la planificación de la materia del viernes aparece QMP3 pero no encontré ningún link al enunciado en la bitácora ni en el aula, aunque sí lo encontré en la bitácora del curso de los jueves.

También, un detalle: veo que el título del formulario de entrega que aparece en el curso de los jueves dice "¡Entregá tu solución de Que Me Pongo **Cuarta** Iteración!" aunque sí tiene el link a la tercera.

Saludos

**Comment by gastonprieto** [comment, marked as answer]
Buenas @RaniAgus 

Ya corregí el titulo para lo de los jueves.

Y te paso el link de QMP3: https://docs.google.com/document/d/1th_F_BsF91zKuFWk6POogegljhrVW9ThybW_9U_xJZY/edit

**Reply by RaniAgus** [reply]
Graciaas 🤗
